### DISCUSSION TITLE
Certain countries can't access?

### DISCUSSION THREAD

**Discussion by nicktan147** [WalletConnect/walletconnect-monorepo#2958] [Developer Support]
I have upgraded successfully to v2, we tried on Malaysia it's working, however most of the China users are facing issue.
![IMG_20230703_192040](https://github.com/WalletConnect/walletconnect-monorepo/assets/7764638/3d80652d-f7da-4965-946a-364861afabce)

I can't seems to find what is the problem. Im using Ethereum Provider for this.

**Comment by ThianHooi** [comment]
@nicktan147 Have you found any workaround for this issue?
